project.godot

```diff
@@ -96,11 +96,6 @@ interact={
 2d_physics/layer_1="normal"
 2d_physics/layer_2="grapple"
 
-[layer_names]
-
-2d_physics/layer_1="normal"
-2d_physics/layer_2="grapple"
-
```

Comment:
@DaltonSW is this something you added? Seems like the important part might actually be above

Reply:
Confirmed off GitHub that this isn't needed